app/grandchallenge/products/views.py

```diff
@@ -191,6 +196,17 @@ class ContactPage(TemplateView):
     template_name = "products/contact.html"
 
 
+class ProjectAirPage(ListView):
+    template_name = "products/project_air.html"
+    model = ProjectAirFiles
+    context_object_name = "project_air_files"
+    queryset = ProjectAirFiles.objects.all()
+
+    def get_context_data(self, *args, **kwargs):
+        context = super().get_context_data(*args, **kwargs)
+        return context
```

Comment:
This doesn't do anything and can be removed

Reply:
done